userspace/libsinsp/tuples.cpp

```diff
@@ -50,3 +64,62 @@ bool ipv6addr::in_subnet(const ipv6addr &other) const
 	return (m_b[0] == other.m_b[0] &&
 		m_b[1] == other.m_b[1]);
 }
+
+void ipv6net::init(const std::string &str)
+{
+	std::stringstream ss(str);
+	std::string ip, mask;
+
+	getline(ss, ip, '/');
+	getline(ss, mask);
+
+	if(inet_pton(AF_INET6, ip.c_str(), m_addr.m_b) != 1)
+	{
+		throw sinsp_exception("unrecognized IPv6 address " + std::string(str));
+	}
+
+	uint32_t prefix_len = sinsp_numparser::parseu8(mask);
+
+	if (prefix_len == 0 || prefix_len > 128)
+	{
+		throw sinsp_exception("invalid v6 netmask " + mask);
+	}
+
+	m_mask_len_bytes = prefix_len / 8;
+	m_mask_tail_bits = 8 - prefix_len % 8;
```

Comment:
nit, for readability:
```suggestion
	m_mask_tail_bits = 8 - (prefix_len % 8);
```